templates/template_functions_test.go

```diff
@@ -588,6 +588,28 @@ func TestContains(t *testing.T) {
 	}
 }
 
+func testTorrentFileExists(t *testing.T) {
+	var tests = []struct {
+		hash 	     string
+		Expected     bool
+	}{
+		{
+			hash: "98C5B0FC5FC996A565D5DFE2312C467213BCC2EB",
+			Expected: true,
```

Comment:
You can't test "true" since you don't provide the torrent with hash "98C5B0FC5FC996A565D5DFE2312C467213BCC2EB" in the source code, so the torrent doesn't exist.
You should test "" return false. And I think it's enough.